src/serlo-editor/plugins/input-exercise/editor.tsx

```diff
@@ -90,7 +90,7 @@ export function InputExerciseEditor(props: InputExerciseProps) {
               setTimeout(() => {
                 dispatch(focus(id))
                 newestAnswerRef.current?.focus()
-              }, 10)
+              }, 50)
```

Comment:
can you add a comment why you increased the timeout? (or point out that it's arbitrary xD)

Reply:
oh thanks, I had exactly that comment but it got lost it seems.